symfony/client/src/components/dialogs/DeleteConfirmationDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <oxd-dialog v-model:show="show" :style="{ maxWidth: '450px' }">
+    <div class="orangehrm-modal-header">
+      <oxd-text type="card-title">Are you sure?</oxd-text>
+    </div>
+    <div class="orangehrm-text-center-align">
+      <oxd-text type="subtitle-2">
```

Comment:
can we have better name for `subtitle-2` ?

Reply:
This is predefined in OXD types fro Oxd-Text component. should i change it?